packages/admin-ui/src/Radio/RadioGroupPresenter.ts

```diff
@@ -0,0 +1,48 @@
+import { makeAutoObservable } from "mobx";
+import { RadioGroupVm } from "./RadioGroup";
+import { RadioItem } from "~/Radio/RadioItem";
+import { RadioItemMapper } from "~/Radio/RadioItemMapper";
+import { RadioItemDto } from "~/Radio/RadioItemDto";
+
+interface RadioGroupPresenterParams<TValue = string> {
+    items: RadioItemDto[];
+    onValueChange: (value: TValue) => void;
+}
+
+interface IRadioGroupPresenter<TValue = string> {
+    vm: RadioGroupVm;
+    init: (props: RadioGroupPresenterParams<TValue>) => void;
+    changeValue: (value: TValue) => void;
+}
+
+class RadioGroupPresenter<TValue = string> implements IRadioGroupPresenter<TValue> {
+    private params?: RadioGroupPresenterParams<TValue> = undefined;
+
+    constructor() {
+        makeAutoObservable(this);
+    }
+
+    public init = (params: RadioGroupPresenterParams<TValue>) => {
+        this.params = params;
+    };
+
+    get vm() {
+        return {
+            items: this.items.map(item => RadioItemMapper.toFormatted(item))
```

Comment:
Question I had here before... why is `RadioItemMapper` having the `toFormatted` fn. If it's a "mapper", shouldn't it have `toMapped` method instead?

Don't know what's the correct term/verb here: are we talking about a mapper or a formatter here? 🤔

Reply:
It makes sense: the original idea was to have a `RadioItemMapper` class with multiple methods (`toFormatted`, `toDto`...).

I changed it to `RadioItemFormatter` with a single `format` method.